src/ui/sass/modules/_Pagination.scss

```diff
@@ -55,4 +55,10 @@
   {
     margin-right: -4px;
   }
+
+  &-chevron,
```

Comment:
oh just so I can learn css how come they weren't centered vertically without display: flex?

Reply:
This is an element wrapping the chevron icon, but it was reaching the height of its parent container! `display: flex` tells it to match its child element. So flex isn't necessarily centering here, it's just removing extra space inside the div that's making it off-center.